### PR TITLE
RUL-448: add text and new_line options in concatenate action rule

### PR DESCRIPTION
**Pull request by nmarniesse**
**Description**

Add 2 options in the concatenate action rule:
- `text` to add a specific text
- `new_line` to add a new line / line break

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 1]
```diff
@@ -418,7 +418,7 @@ Two parameters are required while the three others are optional.
 Concatenate
 ___________
 
-This action concatenates at least two values into a single value. A space separates each source value.
+This action concatenates at least two sources into a single value. A source can be an attribute value, a text or a line break.
```
Comment (mmetayer) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst:421]: not sure about "text", to check with Camille

**Review by mmetayer — APPROVED** [review]

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 2]
```diff
@@ -437,14 +437,23 @@ The possible target attribute types are:
...
 |      |                                                                                                                                                                            |
 |      | - field: attribute code.                                                                                                                                                   |
-|      | - locale: locale code for which the value is assigned, for localizable attributes (optional).                                                                              |
-|      | - scope: channel code for which the value is assigned, for scopable attributes (optional).                                                                                 |
+|      | - text: a specific text.                                                                                                                                                   |
+|      | - new_line: no specific value (e.g. ``new_line: ~``). Only available if the target attribute is a text area.                                                               |
```
Comment (pierallard) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst:451]: `new_line: ~` is not an example; it's the only way to do, no ?

**Review by pierallard — APPROVED** [review]

File: manipulate_pim_data/rule/general_information_on_rule_format.rst (hunk 2 above)
Comment (nmarniesse) [reply manipulate_pim_data/rule/general_information_on_rule_format.rst:451]: you can use `new_line: true` or `new_line: "I love raclette"` it will work

**Comment by camillebourjade** [comment]
“English request”
Name: Camille
Size: S
Priority: Major 
Wordcount: 345
Deadline: Sept. 17th

**Review by camillebourjade — COMMENTED** [review]
english fix

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 3]
```diff
@@ -437,14 +437,23 @@ The possible target attribute types are:
  - text
  - textarea
 
+By default a space is added between two source attribute values (not between a source attribute and a source text, or between two source texts). You can avoid this behavior by adding an empty text between the two source attribute values.
```
Comment (mmetayer) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst:440]:
```suggestion
By default, a space is added between two attribute value blocks (not between an attribute and a text block, or between two text blocks). You can avoid this behavior by adding an empty text between the two attribute value blocks.
```

**Review by mmetayer — APPROVED** [review]

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 4]
```diff
@@ -418,9 +418,9 @@ Two parameters are required while the three others are optional.
 Concatenate
 ___________
 
-This action concatenates at least two values into a single value. A space separates each source value.
+This action concatenates at least two blocks into a single value. A block can be an attribute value or a text. You can also use the specific *new_line* block to start a new line.
 
-The possible source attribute types are:
+The possible block attribute types are:
```
Comment (camillebourjade) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst:423]:
```suggestion
The possible attribute types are:
```

**Review by camillebourjade — COMMENTED** [review]
last english check :-)

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 5]
```diff
@@ -418,9 +418,9 @@ Two parameters are required while the three others are optional.
 Concatenate
 ___________
 
-This action concatenates at least two values into a single value. A space separates each source value.
+This action concatenates at least two blocks into a single value. A block can be an attribute value or a text. You can also use the specific *new_line* block to start a new line.
 
-The possible source attribute types are:
+The possible attribute types are:
```
Comment (morganen) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst:423]: Why did you remove the "source" word?

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 6]
```diff
@@ -467,14 +476,17 @@ The possible target attribute types are:
 
 .. tip::
 
-    For instance, to concatenate the brand (non-localizable and non-scopable) and the model in en_US locale into the description value in en_US locale, the action will be as follows:
+    For instance, to concatenate the "Brand: " text, the brand (non-localizable and non-scopable), a line break, the "Model: " text and the model in en_US locale into the description value in en_US locale, the action will be as follows:
```
Comment (morganen) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst:479]:
```suggestion
    For instance, to concatenate the "Brand: " text, the brand attribute (non-localizable and non-scopable), a line break, the "Model: " text and the model attribute in en_US locale into the description value in en_US locale, the action will be as follows:
```

File: manipulate_pim_data/rule/general_information_on_rule_format.rst (hunk 5 above)
Comment (nmarniesse) [reply manipulate_pim_data/rule/general_information_on_rule_format.rst:423]: Camille B wanted to
